docs/contributing.rst

```diff
@@ -90,9 +92,17 @@ you have cloned the Zarr source code and your current working directory is the r
 the repository, you can do something like the following::
 
     $ mkdir -p ~/pyenv/zarr-dev
+    
     $ python -m venv ~/pyenv/zarr-dev
+    
     $ source ~/pyenv/zarr-dev/bin/activate
+    
+For Windows, use::
```

Comment:
![Screen Shot 2022-10-14 at 16 58 06](https://user-images.githubusercontent.com/88113/195878270-c61b9a58-4731-4ed6-a6ba-d9f2e37e6c19.png)
This makes it look like the next three lines are for Windows only rather than one needs "Scripts" rather than "bin" on Windows. Maybe duplicate the entire block for Windows? (And remove the `$` since that is a Unix prompt)

Reply:
Thanks @joshmoore. For some reason, I never got the notification for this reply. Let me get on it right away.

Reply:
@joshmoore Done. Pls review.